library/styleguideComponents/ComponentsList/index.js

```diff
@@ -5,13 +5,14 @@ import ExperiencesIcon from './icons/experiencesIcon';
 import PuzzleIcon from './icons/puzzleIcon';
 import StartupIcon from './icons/startupIcon';
 import DesignIcon from './icons/designIcon';
+import ConsoleIcon from './icons/consoleIcon';
 import isNotComponentPage from './../utils';
 
 const name = window.location.href.split('#/')[1];
 const pageName = name && name.split('/')[0];
 
 function resolveCurrentUrl() {
-    if (isNotComponentPage(pageName)) {
+    if (isNotComponentPage(pageName) || pageName === 'Customization') {
```

Comment:
we should add 'Customization' to the pages array inside isNotComponentPage function